app/code/Magento/Catalog/view/frontend/templates/product/view/form.phtml

```diff
@@ -30,11 +30,17 @@
         <?php else :?>
             <?php if ($_product->isSaleable() && $block->getOptionsContainer() == 'container1') :?>
                 <?= $block->getChildChildHtml('options_container') ?>
+            <?php elseif ($_product->getTypeId() == 'downloadable' &&
```

Comment:
All changes related to the Downloadable product should be int the Downloadable module instead of Catalog